pylint/lint/__init__.py

```diff
@@ -87,5 +87,18 @@
     preprocess_options,
 )
 
+__all__ = [
```

Comment:
@Pierre-Sassoulas according to pep8 `__all__` flags are supposed to go between future imports and other imports

Reply:
https://github.com/PyCQA/isort/issues/711

Reply:
I'm going to merge because we can fix the `__all__` in another MR (probably when isort does it automatically) and benefit from the auto-fixes of autoflake until then.